.gitignore

```diff
@@ -9,3 +9,5 @@ npm-debug.log
 // vim .swp
 *.swp
 .env
+
+.idea/
```

Comment:
sorry I didn't get what do you mean... It's because of the extra line ? So it shouldn't have a new line (on line 12) ?